awx/api/serializers.py

```diff
@@ -1408,11 +1402,7 @@ def get_field_from_model_or_attrs(fd):
                             ' '.join([str(pk) for pk in used_by])
                         )})
 
-        view = self.context.get('view', None)
-        if not organization and not view.request.user.is_superuser:
-            # Only allow super users to create orgless projects
-            raise serializers.ValidationError(_('Organization is missing'))
```

Comment:
some version of this may need to be put back in

Reply:
Is this taken care of anywhere else in here?